noxfile.py

```diff
@@ -129,6 +132,31 @@ def get_plugin_python_version(session, plugin):
     )
 
 
+def get_plugin_os_names(session: nox.session, plugin: str) -> List[str]:
+    setup_py = os.path.join(BASE, "plugins", plugin["path"], "setup.py")
+    out = session.run("python", setup_py, "--classifiers", silent=True).splitlines()
+    oses = list(filter(lambda line: "Operating System" in line, out))
+    if len(oses) == 1 and oses[0] == "Operating System :: OS Independent":
```

Comment:
What if OS is not specified at all?

Reply:
Testing for all OS now

Reply:
Yes, you are right. I have fixed this in the new commit. The comment `Testing for all OS now` is the behavior after the new commit. I should have been more explicit :)

Reply:
Fixed in https://github.com/facebookresearch/hydra/pull/434/commits/f1a5692a97a3f5a3a172f48f27e3bfaba7f1f94a